cmd/siac/rentercmd.go

```diff
@@ -475,11 +477,11 @@ func rentercontractscmd() {
 // rentercontractsviewcmd is the handler for the command `siac renter contracts <id>`.
 // It lists details of a specific contract.
 func rentercontractsviewcmd(cid string) {
-	rc, err := httpClient.RenterContractsGet()
+	rc, err := httpClient.RenterContractsGet(true)
```

Comment:
since it's not obvious at all what `true` means in the context of this function, convention is to use a variable:

```
getAllContracts := true
rc, err := httpCliend.RenterContractsGet(getAllContracts)
```